WORKSPACE

```diff
@@ -0,0 +1,134 @@
+# gazelle:repository_macro hack/build/repos.bzl%go_repositories
+workspace(name = "com_github_coachroachdb_cockroach_operator")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+http_archive(
+    name = "com_google_protobuf",
+    sha256 = "2ee9dcec820352671eb83e081295ba43f7a4157181dad549024d7070d079cf65",
+    strip_prefix = "protobuf-3.9.0",
+    urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.9.0.tar.gz"],
+)
+
+load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
+
+protobuf_deps()
+
+## Load rules_go and dependencies
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "a8d6b1b354d371a646d2f7927319974e0f9e52f73a2452d2b3877118169eb6bb",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.23.3/rules_go-v0.23.3.tar.gz",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.23.3/rules_go-v0.23.3.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+
+go_rules_dependencies()
+
+go_register_toolchains(
+    go_version = "1.14.4",
+    # todo nogo is failing
+    # nogo = "@//hack/build:nogo_vet",
```

Comment:
Ran into the same issues in CC, stumbled across these:

https://github.com/bazelbuild/rules_go/issues/2513
https://github.com/bazelbuild/rules_go/issues/2396

Reply:
So can this work?